snmp/datadog_checks/snmp/data/default_profiles/cyberpower-pdu.yaml

```diff
@@ -0,0 +1,375 @@
+    symbols:
+      - name: cyberpower.ePDULoadBankConfig
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.3.2.4.1.1.1
+          name: ePDULoadBankConfigIndex
+        tag: e_pdu_load_bank_config_index
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.3.2.4.1.1.5
+          name: ePDULoadBankConfigAlarm
+        tag: e_pdu_load_bank_config_alarm
+        mapping:
+          1: no_load_alarm
+          2: under_current_alarm
+          3: near_over_current_alarm
+          4: over_current_alarm
+  - MIB: CPS-MIB
+    table:
+      OID: 1.3.6.1.4.1.3808.1.1.3.3.5.1
+      name: ePDUOutletStatusTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3808.1.1.3.3.5.1.1.7
+        name: cyberpower.ePDUOutletStatusLoad
+      - OID: 1.3.6.1.4.1.3808.1.1.3.3.5.1.1.8
+        name: cyberpower.ePDUOutletStatusActivePower
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.3.3.5.1.1.1
+          name: ePDUOutletStatusIndex
+        tag: e_pdu_outlet_status_index
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.3.3.5.1.1.2
+          name: ePDUOutletStatusOutletName
+        tag: e_pdu_outlet_status_outlet_name
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.3.3.5.1.1.4
+          name: ePDUOutletStatusOutletState
+        tag: e_pdu_outlet_status_outlet_state
+        mapping:
+          1: outlet_status_on
+          2: outlet_status_off
+      - column:
+          OID: 1.3.6.1.4.1.3808.1.1.3.3.5.1.1.9
+          name: ePDUOutletStatusAlarm
+        tag: e_pdu_outlet_status_alarm
+        mapping:
+          1: no_load_alarm
+          2: under_current_alarm
+          3: near_over_current_alarm
+          4: over_current_alarm
+  # - MIB: CPS-MIB  # All three metric tags are scalar metrics with enums. Not supported yet.
+  #   table:
+  #     OID: 1.3.6.1.4.1.3808.1.1.3.3.6.1
+  #     name: ePDUOutletBankTable
+  #   symbols:
+  #     - name: ePDUOutletBank
+  #       constant_value_one: true
+  #   metric_tags:
+  #     - column:
+  #         OID: 
+  #         name: ePDUOutletBankIndex
+  #     - column:
+  #         OID: 1.3.6.1.4.1.3808.1.1.3.4.1.1
+  #         name: ePDUPowerSupply1Status
+  #       tag: e_pdu_power_supply1_status
+  #       mapping:
+  #         1: power_supply_one_ok
+  #         2: power_supply_one_failed
+  #     - column:
+  #         OID: 1.3.6.1.4.1.3808.1.1.3.4.1.2
+  #         name: ePDUPowerSupply2Status
+  #       tag: e_pdu_power_supply2_status
+  #       mapping:
+  #         1: power_supply_two_ok
+  #         2: power_supply_two_failed
+  #         3: power_supply_two_not_present
+  #     - column:
+  #         OID: 1.3.6.1.4.1.3808.1.1.3.4.1.3
+  #         name: ePDUPowerSupplyAlarm
+  #       tag: e_pdu_power_supply_alarm
+  #       mapping:
+  #         1: all_available_power_supplies_ok
+  #         2: power_supply_one_failed
+  #         3: power_supply_two_failed
+  #         4: power_supply_oneand_two_failed
```

Comment:
I'm unclear on what makes the commented-out block above different from the block below - why do we have to exclude the outlet bank table but not the status bank table?

Reply:
Oh right, this [comment](https://github.com/DataDog/integrations-core/pull/15158/files/aecf6d51991513115c7a013dcae6b8ce566e7f8f#diff-31bc1ea9605608da6582b63eae6e3097c0926269a10e8e3b8d4ada3c2497c777R97) seems wrong.
I don't remember the details that might be a mistake indeed